xml/System.Diagnostics.Eventing.Reader/EventLogReader.xml

```diff
@@ -464,16 +464,52 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Reads the next event that is returned from the event query in this object.</summary>
-        <returns>Returns an <see cref="T:System.Diagnostics.Eventing.Reader.EventRecord" /> object.</returns>
+        <summary>Reads and returns the next event from the event query results - or null if the end of the results has been reached.</summary>
```

Comment:
```suggestion
        <summary>Reads and returns the next event from the event query results.</summary>
```

Reply:
Methinks I should add the `<returns>` element back…

…but why remove the important details about `null` here?